File: LongestConsecutiveSequence/elixit/solution_test.exs
Code.require_file("./solution.exs")

ExUnit.start()

defmodule LongestConsecutiveSequenceTest do
  use ExUnit.Case

  test "empty" do
    assert 0 == LongestConsecutiveSequence.longestConsecutive([])
  end

  test "unordered" do
    assert 4 == LongestConsecutiveSequence.longestConsecutive([100, 4, 200, 1, 3, 2])
  end
end

File: LongestConsecutiveSequence/java/SolutionTest.java
package LongestConsecutiveSequence.java;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

public class SolutionTest {
    @Test
    void empty() {
        assertEquals(0, Solution.longestConsecutive(new int[] { }));
    }

    @Test
    void unordered() {
        assertEquals(4, Solution.longestConsecutive(new int[] { 100, 4, 200, 1, 3, 2 }));
    }
}
